data/successors/successor 1 prologue.txt

```diff
@@ -365,6 +365,16 @@ mission "Successors: First Contact 3"
 			choice
 				`	"We'd better get started, then."`
 					goto embark
+				`	"What do you mean that jump drives are 'familiar?'"`
```

Comment:
I've played through this using the provided save and everything seems to work as expected. Thank you for addressing this!

My one suggestion is to change the tense for the jump drives question here, since the player may not longer be asking it as an immediate follow-up.
```suggestion
				`	"What did you mean that jump drives are 'familiar?'"`
```